pkgs/development/python-modules/jaxlib/default.nix

```diff
@@ -117,6 +123,8 @@ let
     ] ++ lib.optionals cudaSupport [
       cudatoolkit
       cudnn
+    ] ++ lib.optionals (!stdenv.isDarwin) [
+      nsync
     ];
```

Comment:
I was under the impression that the Darwin build was being disabled until we get the macOS 10.14 SDKs?

Reply:
This came from syncing with the tensorflow derivation (here: https://github.com/NixOS/nixpkgs/blob/3524da2f0ab911c9c65ac6fb34e87b9698a4e5fb/pkgs/development/python-modules/tensorflow/default.nix#L232-L234)
My thought was to leave the darwin-specific workarounds in place (even though the build is disabled for now) in case switching to the newer SDK fixes the remaining issues. I'm not really sure of the recommended practice in this situation, though -- happy to remove the gating if that seems preferable.

Reply:
Gotcha, yeah now that we will have darwin builds working soon (shout out to @uri-canva!), we can figure how to make this work after all